polaris.shopify.com/content/components/inline.md

```diff
@@ -10,12 +10,27 @@ status:
 examples:
   - fileName: inline-default.tsx
     title: Default
-  - fileName: inline-with-align.tsx
-    title: Horizontal alignment
     description: >-
-      Horizontal alignment for children can be set with the align property.
+      By default, Inline is left aligned, and wraps based on the width of Content block or Page. The default spacing in between is 16px and Inline children are horizontally centered.
```

Comment:
```suggestion
      By default, Inline is left aligned and wraps based on the width of the surrounding layout. Items are horizontally centered and the spacing around them is 16px.
```

Reply:
Do we need this description? It's pretty repetitive to the description. I don't think we need to be so detailed here either so I'm not sure if this adds value or not?

Reply:
I agree, I'm a fan of the shorter description for this one 👍🏾